internal/client/integrations/integrations.go

```diff
@@ -507,15 +520,16 @@ func Get(name string, version string, basicInfo bool, minimal bool, override boo
 		}
 
 		var or overrides
-
+		var respOvrBody []byte
 		if or, err = extractOverrides(iversion); err != nil {
 			return nil, err
 		}
-		if respBody, err = json.Marshal(or); err != nil {
+		if respOvrBody, err = json.Marshal(or); err != nil {
 			return nil, err
 		}
 		apiclient.ClientPrintHttpResponse.Set(apiclient.GetCmdPrintHttpResponseSetting())
-		apiclient.PrettyPrint(respBody)
+		apiclient.PrettyPrint(respOvrBody)
+		return respBody, err
```

Comment:
logic here is causing the overrides file to be replaced with the contents of the integration json itself